tariff/octopus.go

```diff
@@ -96,24 +99,30 @@ func (t *Octopus) run(done chan error) {
 
 	// If ApiKey is available, use GraphQL to get appropriate tariff code before entering execution loop.
 	if t.apikey != "" {
-		gqlCli, err := octoGql.NewClient(t.log, t.apikey)
+		t.log.TRACE.Println("running in GraphQL mode")
+		gqlCli, err := octoGql.NewClient(t.log, t.apikey, t.accountnumber)
 		if err != nil {
 			once.Do(func() { done <- err })
 			t.log.ERROR.Println(err)
 			return
 		}
+		t.log.TRACE.Println("requesting tariff code")
```

Comment:
I'd like to get rid of this logging before merge. The graphql queries should already be visible in the trace log.

Reply:
9fed9f5

Reply:
I mean, all of them once you're satisfied that it works ;) Trace log in evcc is typically user for the physical transports, not for commenting program logic.